docs/usermanual.md

```diff
@@ -544,3 +545,95 @@ To ensure consistent Markdown formatting run the following:
 # Use git-bash on Windows
 npm run prettier:text
 ```
+
+### Swagger
+
+In order to run Swagger, you need to execute the
+[IoT Agent](https://github.com/telefonicaid/iotagent-ul/blob/master/docs/installationguide.md#usage) and then you can
+access to:
+
+```
+<server_host>:7896/api-docs
```

Comment:
One doubt...

IOTA exposes two ports: one corresponding to the "southbound interface" (to which the devices send measures and command responses) and another corresponding to the "nourthbound interface" (in which the IOTA provides the provisioning API, i.e. to create a new device, etc.).

Does this /api-docs endpoint work in both ports? Or only in northbound/southbound?

Reply:
we have developed the Swagger documentation only for the HTTP protocol (https://fiware-iotagent-ul.readthedocs.io/en/latest/usermanual/index.html#http-binding), so this version of swagger documentation (1.0.0) works only in southbound.

Reply:
Northbound is also HTTP-based...

However, if /api-docs only works in southbound, fine with it. I just want to confirm how it works ;)

Reply:
This documentation has been based on the sending of measurements using exclusively the HTTP protocol, so works in southbound.

Reply:
fixed in 2164dc01ef8494fe5ab99c5da6956cd8fa9b9094